tests/integration_tests/style/test_licenses.py

```diff
@@ -90,16 +88,10 @@ def test_for_valid_licenses():
     """
     Test that all *.py, *.rs and *.sh files contain a valid license.
     """
-    python_files = utils.get_files_from(
-        find_path="..", pattern="*.py", exclude_names=EXCLUDE
-    )
```

Comment:
I think this eliminates all uses of `utils.get_files_from`, should we delete it?

Reply:
Good point. Removing.